<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KileWidgetHelpConfig</class>
 <widget class="QWidget" name="KileWidgetHelpConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>358</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Help</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="m_lbLocation">
       <property name="whatsThis">
        <string>Insert the path to the TeX documentation directory here. For example /usr/share/texmf/doc.</string>
       </property>
       <property name="text">
        <string>&amp;Location of TeX documentation:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>kcfg_location</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="kcfg_location"/>
     </item>
     <item>
         <widget class="KPushButton" name="m_helpLocationButton"/>
        </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="m_gbContextHelp">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Context Sensitive Help</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QRadioButton" name="kcfg_kilerefs">
        <property name="text">
         <string>Use the &amp;Kile LaTeX reference</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="buttonGroupId" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="kcfg_texrefs">
        <property name="text">
         <string>Use your system's &amp;TeX documentation</string>
        </property>
        <property name="buttonGroupId" stdset="0">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_gbUserHelp">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>User Help</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="m_pbConfigure">
          <property name="text">
           <string>Con&amp;figure...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="kcfg_embedded">
        <property name="text">
         <string>Use &amp;embedded viewer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="kcfg_external">
        <property name="text">
         <string>Show help file in a &amp;separate window</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">klineedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
